src/liiteri/db/file_metadata_store.clj

```diff
@@ -19,11 +19,17 @@
                              {:connection conn-arg#})]
      ~@body))
 
+(def ^:private non-valid-character-pattern #"(?i)[^a-z\.\-_0-9 ]")
+
+(defn- normalize [string]
+  (clojure.string/replace string non-valid-character-pattern ""))
```

Comment:
Does this replace _every_ occurrence of evil characters in the string, or just the first?